english/count-the-marshmellows/eat-all-the-marshmellows/pillow-problems/pillow-problems.md

```diff
@@ -9,4 +9,6 @@ itself, now reduced to shards? You fumble through its remnants and find the tag:
 experimental material.  Not for consumption.  If swallowed, call 1-888-285-20"  Wait, "experimental?" 
 You've chewed off the last 2 digits, and your stomach isn't feeling so right suddenly.  Now what?
 You recall that California is in the midst of a drought, so what will help is a nice large glass
-of water. You survey the room again, looking for something cool and refreshing.
\ No newline at end of file
+of water. You survey the room again, looking for something cool and refreshing.
+
+You spot a bathtub filled with hot chocolate. It won't be cool or refreshing, but you don't have time. You lean over it, cup your hand, and drink rapidly. Your buddy looks at you incredulously. "What?" you ask nervously.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.